lms/envs/common.py

```diff
@@ -2274,9 +2274,7 @@
         'base_url': os.path.join(MEDIA_URL, 'profile-images/'),
     },
 }
-PROFILE_IMAGE_DEFAULT_FILENAME = (
-    'images/edx-theme/default-profile' if FEATURES['IS_EDX_DOMAIN'] else 'images/default-theme/default-profile'
```

Comment:
This should default to the value in common.py, e.g. `PROFILE_IMAGE_DEFAULT_FILENAME = PROFILE_IMAGE_DEFAULT_FILENAME if FEATURES['IS_EDX_DOMAIN'] else 'images/default-theme/default-profile'`